recipes/pycapnp/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.5.7" %}
+
+package:
+  name: pycapnp
+  version: {{ version }}
+
+source:
+  fn: pycapnp-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pycapnp/pycapnp-{{ version }}.tar.gz
+  md5: d47396d6724593afcbb728b87ac6de07
+
+build:
+  # pycapnp does not support Windows due to the lack of dynamic API support in
+  # C++ Cap'n Proto on Windows:
+  # https://github.com/jparyani/pycapnp/issues/53
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - capnproto
+
+  run:
+    - python
+
+test:
+  imports:
+    - capnp
+
+about:
+  home: http://jparyani.github.io/pycapnp/
+  license: BSD 2-clause
+  summary: A python wrapping of the C++ implementation of the Cap’n Proto library
```

Comment:
To "fix" the linter you should change `’` to `'` and add wrap that in `""`. However, to truly fix the linter, we should change it to parse `unicode` :smile: